packages/flutter_tools/lib/src/flx.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 
+import 'package:flutter_tools/src/artifacts.dart';
```

Comment:
you import this already lower down

in any case, never import via package: to the current package